packages/fx-core/src/component/resource/azureSql/actions/configure.ts

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { hooks } from "@feathersjs/hooks/lib";
+import {
+  ContextV3,
+  InputsWithProjectPath,
+  ok,
+  FunctionAction,
+  ProvisionContextV3,
+  v3,
+  AzureAccountProvider,
+  LogProvider,
+  Result,
+  FxError,
+  Effect,
+} from "@microsoft/teamsfx-api";
+import { ComponentNames, ActionNames, ActionTypeFunction } from "../../../constants";
+import { ProgressBarMW } from "../../../middleware/progressbar";
+import { ManagementClient } from "../clients/management";
+import { SqlClient } from "../clients/sql";
+import { LoadManagementConfig, LoadSqlConfig } from "../config";
+import { Constants, HelpLinks } from "../constants";
+import { ErrorMessage } from "../errors";
+import { SqlResultFactory } from "../results";
+import { parseToken, TokenInfo, UserType } from "../utils/common";
+import { Message } from "../utils/message";
+
+export function GetActionConfigure(): FunctionAction {
+  return {
+    name: `${ComponentNames.AzureSQL}.${ActionNames.configure}`,
+    type: ActionTypeFunction,
+    plan: (context: ContextV3, inputs: InputsWithProjectPath) => {
+      return ok([{ type: "service", name: "azure", remarks: "configure azure-sql" }]);
+    },
+    execute: ConfigureActionImplement.execute,
+  };
+}
+
+export class ConfigureActionImplement {
+  static readonly stage = "post-provision";
+  static readonly progressTitle: string = "Configuring SQL";
+  static readonly progressMessage = {
+    addAadmin: "Configure aad admin for SQL",
+    addUser: "Configure database user",
+  };
+  @hooks([
+    // RunWithCatchErrorMW(ConfigureActionImplement.stage),
+    ProgressBarMW(
```

Comment:
there're 2 steps but seems no code controls when to show `Configure aad admin for SQL` and when to show `Configure database user`?

Reply:
Thanks for the reminder.
It will be added in the following commit.